### PR TITLE
Document: Fixed required space check always including swap

### PR DESCRIPTION
**Pull request by OndrejZobal**
documented changes in commmit: 8e16a35f933eb7ef5401179aa70b46d9bf8df393

https://github.com/rhinstaller/anaconda/pull/4238

### CODE REVIEW

File: docs/release-notes/swap-estimate-fix.rst [hunk 1]
```diff
@@ -0,0 +1,13 @@
+:Type: GUI
+:Summary: Do not include swap in space estimate if swap is disabled (#2068290)
+
+:Description:
+   During automatic partitioning the disk spoke estimates the space required for the installation
+   and if there isn't enought free space it dispalys a warning dialog suggesting more space should
```
Comment (poncovka) [inline docs/release-notes/swap-estimate-fix.rst:6]: There are some typos.

**Review by poncovka — COMMENTED** [review]
Otherwise, it looks good to me.

**Review by VladimirSlavik — APPROVED** [review]
Thank you!

**Review by jkonecny12 — APPROVED** [review]

**Review by poncovka — APPROVED** [review]
Thank you!

**Pull request merged**